### PR TITLE
add errors in index

### PR DESCRIPTION
**Pull request by tob**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
 <body>
     <header>
         <h1>Tobia was here</h1>
+        <h2></h2>
+        <al>
+            <li>Douwy</li>
+            <li>Ouwey</li>
+            <li>Pewy</li>
+        </al>
```
Comment (Genyus) [inline index.html]:
```suggestion
        <ol>
             <li>Douwy</li>
             <li>Ouwey</li>
             <li>Pewy</li>
         </ol>
```

**Review by Genyus — CHANGES_REQUESTED** [review]
`<al></al>` should be `<ol></ol>`

File: index.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
 <body>
     <header>
         <h1>Tobia was here</h1>
+        <h2></h2>
+        <al>
+            <li>Douwy</li>
```
Comment (Cheulie) [inline index.html]: <li> Hughey <\li>

File: index.html [hunk 3]
```diff
@@ -9,6 +9,12 @@
 <body>
     <header>
         <h1>Tobia was here</h1>
+        <h2></h2>
```
Comment (Damidee1) [inline index.html]: adding title

**Review by Damidee1 — COMMENTED** [review]
add a title

File: index.html (hunk 3 above)
Comment (Damidee1) [reply index.html]: page title please, 'pull request test'

File: index.html (hunk 2 above)
Comment (ghost) [inline index.html]: Let's change this name to Douwey? I don't know their names either :)

Comment (tob) [reply index.html]:
```
<ol>
  <li>tuiwy</li>
</ol>
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (Damidee1) [inline index.html]: title 'pull request test'

File: index.html (hunk 2 above)
Comment (tob) [reply index.html]: Perfect, thanks

**Review by Genyus — APPROVED** [review]
Looks good to me

**Review by Cheulie — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Perfecto!!

**Review by Damidee1 — APPROVED** [review]
good job.

**Pull request merged**